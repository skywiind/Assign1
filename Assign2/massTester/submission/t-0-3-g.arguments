-g t-0-3.corpus t-0-3.commands
