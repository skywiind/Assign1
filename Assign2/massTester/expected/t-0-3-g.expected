0: n<3>(n<3>)X
1: av(n<3>)L nhu(n<3>)R
2: a(av)L it(av)R xh<2>(nhu)R
3: ag(a)R e<2>(it)L mr(it)R s(xh<2>)L ye(xh<2>)R
4: aa(ag)L btg(e<2>)L ig(e<2>)R me(mr)L r<3>(s)L tjc(s)R =xqs(ye)L z<2>(ye)R
5: =adx(aa)R b(btg)L d t(btg)R et(ig)L ip(ig)R m<2>(me)L mq(me)R q<2>(r<3>)L =rs(r<3>)R sjd(tjc)L v<2>(tjc)R yv(z<2>)L zac(z<2>)R
6: =az(b)L =bgc(b)R =d(d t)L ec(et)L hs(et)R =im(ip)L =iq(ip)R k(m<2>)L m u(m<2>)R =mj(mq)L no(q<2>)L qa(q<2>)R =t(sjd)R u(v<2>)L wx(v<2>)R =yzv(yv)R zf(zac)R
7: =ee(ec)R h(hs)L jg(k)L kl(k)R =mbf(m u)R p<3>(no)R =qqm(qa)R =tp(u)L =ujr(u)R va(wx)L =x<2>(wx)R =zrl(zf)R
8: go(h)L =j(jg)L lma(kl)R o(p<3>)L pij(p<3>)R war(va)R
9: ev(go)L =l(lma)L =lvk(lma)R ote(o)R =pl(pij)R w(war)L
10: gh(ev)R =ono(ote)L =vj(w)L
11: =fe(gh)L
Duplicates: 13
Nodes: 77
Minimum depth: 1
Maximum depth: 11
Value jk not found.
Value cf not found.
Value vu not found.
Value gov not found.
Value rt not found.
Value ek not found.
Value hn not found.
Value g not found.
Value id not found.
Value wr not found.
Value c not found.
Value ttx not found.
Value co not found.
Value xs not found.
Value lzw not found.
Value ut not found.
Value pc not found.
Value cvw not found.
Value kyh not found.
Value azs not found.
0: n<3>(n<3>)X
1: av(n<3>)L nhu(n<3>)R
2: a(av)L it(av)R xh<2>(nhu)R
3: ag(a)R e<2>(it)L mr(it)R s(xh<2>)L ye(xh<2>)R
4: aa(ag)L btg(e<2>)L ig(e<2>)R me(mr)L r<3>(s)L tjc(s)R =xqs(ye)L z<2>(ye)R
5: =adx(aa)R az(btg)L d t(btg)R et(ig)L ip(ig)R m<3>(me)L mq(me)R q<2>(r<3>)L =rs(r<3>)R sjd(tjc)L v(tjc)R yv(z<2>)L zac(z<2>)R
6: bgc(az)R d(d t)L ec(et)L hs(et)R =im(ip)L =iq(ip)R jn(m<3>)L m u(m<3>)R mj(mq)L no(q<2>)L qa(q<2>)R =sg(sjd)L =t(sjd)R u(v)L wx(v)R =yzv(yv)R zf(zac)R
7: =b(bgc)L =bzn(d)L =ee(ec)R h(hs)L ic(hs)R =jg(jn)L kl(jn)R =mbf(m u)R =mki(mj)R =nj(no)L p<4>(no)R =qqm(qa)R tp(u)L ujr(u)R va(wx)L x<2>(wx)R =ze(zf)L zrl(zf)R
8: go(h)L =id(ic)R lma(kl)R o<2>(p<4>)L pij(p<4>)R =tm(tp)L =uk(ujr)R war(va)R =wy(x<2>)L =zz(zrl)R
9: ev(go)L =l(lma)L =lvk(lma)R ote(o<2>)R =pl(pij)R w(war)L =wgw(war)R
10: gh(ev)R =ono(ote)L vj(w)L
11: fe(gh)L =vd(vj)L
12: fg(fe)R
13: =gc(fg)R
Duplicates: 15
Nodes: 91
Minimum depth: 1
Maximum depth: 13
Value kb not found.
Value ix not found.
Value ll not found.
Value bf not found.
Value wu not found.
Value i not found.
Value tw not found.
Value hgk not found.
Value mz not found.
Value nxi not found.
Value wj not found.
Value qv not found.
Value elm not found.
Value s not found.
Value jh not found.
Value ia not found.
Value vos not found.
Value ya not found.
Value sc not found.
Value quk not found.
Value nf not found.
Value op not found.
Value xfy not found.
Value vh not found.
0: n<4>(n<4>)X
1: av(n<4>)L nhu(n<4>)R
2: a(av)L it(av)R xh<2>(nhu)R
3: ag(a)R e<2>(it)L mr(it)R rs(xh<2>)L ye(xh<2>)R
4: aa(ag)L btg(e<2>)L ig(e<2>)R me(mr)L r(rs)L tjc(rs)R xqs(ye)L z<2>(ye)R
5: =adx(aa)R az(btg)L d t(btg)R et(ig)L ip(ig)R m<2>(me)L mki(me)R q<2>(r)L sjd(tjc)L v(tjc)R =xm(xqs)L yv(z<2>)L zac(z<2>)R
6: =bgc(az)R d(d t)L ec(et)L hs(et)R =im(ip)L =iq(ip)R jn(m<2>)L m u(m<2>)R =mj(mki)L no(q<2>)L qa(q<2>)R =sg(sjd)L =t(sjd)R u<2>(v)L wx(v)R yzv(yv)R zf(zac)R
7: =bzn(d)L ee(ec)R h(hs)L ic(hs)R =jg(jn)L kl(jn)R =mbf(m u)R =nj(no)L p<4>(no)R =qqm(qa)R =tp(u<2>)L ujr(u<2>)R va(wx)L wy(wx)R =yx(yzv)L =ze(zf)L zrl(zf)R
8: =eg(ee)R go(h)L =hq(h)R =id(ic)R lma(kl)R o<2>(p<4>)L pij(p<4>)R =ub(ujr)L =uk(ujr)R war(va)R =xec(wy)R =zz(zrl)R
9: ev(go)L =l(lma)L =lvk(lma)R ote(o<2>)R =pdt(pij)L =pl(pij)R w(war)L =wgw(war)R
10: gh(ev)R =ono(ote)L vj(w)L
11: fe(gh)L vd(vj)L
12: fg(fe)R =vi(vd)R
13: =gc(fg)R
Duplicates: 13
Nodes: 94
Minimum depth: 1
Maximum depth: 13
